silk/tests/test_integration.py

```diff
@@ -0,0 +1,83 @@
+# (C) Datadog, Inc. 2021-present
```

Comment:
```suggestion
# (C) Datadog, Inc. 2023-present
```

![image](https://user-images.githubusercontent.com/1266346/215043088-cdaa003a-ef16-48b7-9c75-fd6412b0d138.png)

Reply:
Hm this is weird because this file was originally `test_silk.py` and I just renamed the file, but I'll just change it to match and make it easier to compare.